hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/helpers/OmKeyArgs.java

```diff
@@ -256,11 +263,16 @@ public Builder setSortDatanodesInPipeline(boolean sort) {
       return this;
     }
 
+    public Builder setKeyNameList(List<String> keyList) {
+      this.keyNameList = keyList;
+      return this;
+    }
+
     public OmKeyArgs build() {
       return new OmKeyArgs(volumeName, bucketName, keyName, dataSize, type,
```

Comment:
Why do we keep keyName and keyNameList at the same time?

Reply:
I think other operations such as createKey still assume a single KeyName.

Reply:
I will delete keyNameList in KeyArgs, by use list of KeyArgs. To avoid changes to KeyArgs.